src/pybind/mgr/cephadm/templates/services/promtail.yml.j2

```diff
@@ -0,0 +1,20 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: http://{{ client_hostname }}:3100/loki/api/v1/push
+
+scrape_configs:
+- job_name: system
+  static_configs:
+  - targets:
+{% for url in hostnames %}
+    - {{ url }}
+{% endfor %}
+    labels:
+      job: Cluster Logs
+      __path__: /var/log/ceph/**/*.log
```

Comment:
Mmm, that `**` shouldn't be the specific fsid of this cluster? Otherwise, if I have 2 or more clusters on the same machine (something the cephadm allows) this promtail will fetch the logs from all of them, right?